### ISSUE TITLE
Make TEST_TIMEOUT for Chromy adjustable in config.js

### ISSUE THREAD

**Issue by VladimirPittner** [garris/BackstopJS#580]
Hi,

I noticed that when running Chromy, the waitTimeout option is set from constant TEST_TIMEOUT.
```
const TEST_TIMEOUT = 30000;
```
```
let chromy = new Chromy({
    chromeFlags: flags,
    port: port,
    waitTimeout: TEST_TIMEOUT,
    visible: config.debugWindow || false
  }).chain();
```

Would it be possible to pass this option as setting from config? 
Actually it would be totally cool to pass all the options for running Chromy :P through config.js but at least the waitTimeout is kinda important I guess.

Thanks.

**Comment by garris** [comment]
Thanks for posting!  In fact there is a pr for this very feature.  I am just waiting for someone to validate it.  

Could you pull this and let me know if it works.  (I have not tested it yet — so there could be a bug there)  https://github.com/garris/BackstopJS/pull/569


Check the readme for usage notes.

**Comment by garris** [comment]
@VladimirPittner, @kiran-redhat -- many apologies for the last version.  I fixed all the issues -- could you give it another try?

**Comment by VladimirPittner** [comment]
@garris no need for apologies at all ;) ... I have tested your latest update, seems working.

Thanks for the great job!

**Comment by VladimirPittner** [comment]
I guess this can be closed, functionality added in 3.0.32 https://github.com/garris/BackstopJS/issues/584
